src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing_through.cpp

```diff
@@ -48,8 +48,8 @@ void prepare_primitive_fusing_through::run(program& p) {
             if (node->is_type<reshape>() && node->get_dependencies().front().first->is_type<reduce>())
```

Comment:
random spot) nit: could you update error message to have proper spacing?
`Gemmcould not create a primitive descriptor for the matmul primitive` -> `Gemm could not create a primitive descriptor for the matmul primitive`

Reply:
This message comes from onednn. I don't think it can be fixed in this PR.